example/dryrun/branch_cell.hpp

```diff
@@ -110,28 +110,20 @@ arb::cable_cell branch_cell(arb::cell_gid_type gid, const cell_parameters& param
     labels.set("soma", tagged(stag));
     labels.set("dend", tagged(dtag));
 
-    arb::decor decor;
-
-    decor.paint("soma"_lab, arb::density("hh"));
-    decor.paint("dend"_lab, arb::density("pas"));
-
-    decor.set_default(arb::axial_resistivity{100}); // [Ω·cm]
-
-    // Add spike threshold detector at the soma.
-    decor.place(arb::mlocation{0,0}, arb::threshold_detector{10}, "detector");
-
-    // Add a synapse to the mid point of the first dendrite.
-    decor.place(arb::mlocation{0, 0.5}, arb::synapse("expsyn"), "synapse");
+    auto decor = arb::decor()
+        .paint("soma"_lab, arb::density("hh"))
+        .paint("dend"_lab, arb::density("pas"))
+        .set_default(arb::axial_resistivity{100}) // [Ω·cm]
+        // Make a CV between every sample in the sample tree.
+        .set_default(arb::cv_policy_every_segment())
```

Comment:
I guess this is possible, but it doesn't make it easy to read. How about you restart after a break/comment?